_posts/2016-10-05-How-to-extract-ZIP-or-RAR-archives-on-your-iOS-device-for-free.markdown

```diff
@@ -0,0 +1,42 @@
+It truly is a one app fits all file manager. You can transfer files between storage providers, import them into other apps, organize everything in folders and add favorites. It can even play mp3 files in a playlist, a feature Apple still has not added to the stock music app, and play many video formats.
+
+But back to our initial problem of the archive someone sends to you. For this test I downloaded the [Ulysses](https://itunes.apple.com/us/app/ulysses/id950335311?mt=8), the app I do all my writing in, Press Kit from their website. It is a 206 MB ZIP archive full of beautiful images of people writing books in different locations. To download bigger files, I like to use [iCab Mobile](https://itunes.apple.com/en/app/icab-mobile-web-browser/id308111628?mt=8) which excels when it comes to downloading files (This also works with files downloaded in Safari).
+
+![](/img/2016-10-05-Extract-Archives-0.PNG)
+
+Choose to download the archive and it will add it to the integrated  download manager and until the download is finished you can continue browsing.
+
+![](/img/2016-10-05-Extract-Archives-1.PNG)
+
+Once the download is finished you can tap on the archive, where you can choose to extract the file directly in iCab Mobile. But as we want to save the contents in Documents 5 for later use, we choose to select *Open in another app* which will show a share sheet. I had to scroll to the very end of the first section to find Documents 5. You can reorder the apps using the *More* option at the end of first section.
+
+![](/img/2016-10-05-Extract-Archives-2.PNG)
+
+![](/img/2016-10-05-Extract-Archives-3.PNG)
+
+Depending on the size of the file this can take a while to switch to [Documents](https://itunes.apple.com/app/documents-by-readdle/id364901807). It will automatically add the file to your documents directory and you can organize it into your desired folder later. We choose to extract the file by tapping on it once. We are greeted with a *Processing* activity indicator that extracts the archive contents into the current directory.
+
+![](/img/2016-10-05-Extract-Archives-4.PNG)
+
+I had to hurry making a screenshot as the app is lightning fast extracting the archive. It will highlight the extracted files or folder for a few moments and you are free to do whatever you want with the results.
+
+![](/img/2016-10-05-Extract-Archives-5.PNG)
+
+## Conclusion
+
+Documents does a very good job at extracting archives. I tried the same scenario with different file types and it could always extract the files without any problems.
+
+Even if you do not plan to use Documents as your main document provider it still proves to be very helpful with the sheer amount of file types it accepts.
```

Comment:
Well done :)
Just check, sometimes you write in 'we'-form and other times in 'I'-form.